infra/terraform/modules/data_access/users/tests/conftest.py

```diff
@@ -0,0 +1,88 @@
+import json
+import mock
+
+import boto3
+import pytest
+
+
+TEST_SAML_PROVIDER_ARN = "arn:aws:iam::123456789012:saml-provider/auth0"
+TEST_STAGE = "test"
+TEST_USERNAME = "alice"
+
+TEST_ROLE_POLICY_ARN = "test_policy_arn"
+
+
+@pytest.fixture
+def username():
+    return TEST_USERNAME
```

Comment:
Thanks for pointing out how to import these constants from the `conftest.py` file  👍 - I should probably read some Python "theory" :P, I removed some of the fixtures and simplified the tests.